docs/attribution.md

```diff
@@ -101,3 +101,77 @@ DOC/NOAA/NESDIS/NCEI > National Centers for Environmental Information, NESDIS, N
 ```
 
 More details can be found on the ETOPO1 [about](https://www.ngdc.noaa.gov/mgg/global/global.html) page and [data sources](https://www.ngdc.noaa.gov/mgg/global/etopo1sources.html) page, and [description](https://www.ngdc.noaa.gov/docucomp/page?xml=NOAA/NESDIS/NGDC/MGG/DEM/iso/xml/316.xml&view=getDataView&header=none) page and [disclaimer](https://www.ngdc.noaa.gov/ngdcinfo/privacy.html#disclaimer) page.
+
+### Land Information New Zealand (LINZ)
+
+[Released](https://data.linz.govt.nz/layer/1768-nz-8m-digital-elevation-model-2012/) by [LINZ](https://data.linz.govt.nz/), licensed under [Creative Commons Attribution 3.0 New Zealand](https://data.linz.govt.nz/license/attribution-3-0-new-zealand/).
+
+Attribution statement:
+
+```
+Copyright 2011 Crown copyright (c) Land Information New Zealand and the New Zealand Government. All rights reserved
+```
+
+### data.gov.uk LIDAR Composite Digital Terrain Model
+
+[Released](https://data.gov.uk/dataset/lidar-composite-dtm-2m1) by [data.gov.uk](https://data.gov.uk/), licensed under [Open Government License version 3](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)
+
+Attribution statement:
+
+```
+© Environment Agency copyright and/or database right 2015. All rights reserved.
+```
+
+### data.gv.at Digitales Geländemodell (DGM) Österreich
+
+[Released](https://www.data.gv.at/katalog/dataset/b5de6975-417b-4320-afdb-eb2a9e2a1dbf) by [data.gv.at](https://www.data.gv.at/), licensed under [Creative Commons Namensnennung 3.0 Österreich](https://creativecommons.org/licenses/by/3.0/at/deed.de)
+
```

Comment:
I don't see a specific statement they require as attribution. The ([English translation of the](https://creativecommons.org/licenses/by/3.0/at/deed.en)) license says:

> You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.

All the other "attribution statements" in this document are verbatim from the license or metadata, so it's odd to generate one ourselves and include it.